packages/core/template/default/404.md

```diff
@@ -0,0 +1,14 @@
+<frontmatter>
+    title: Page not found
+    layout: 404.md
+</frontmatter>
+
+<body style="font-family: monospace; background-color: #f1f1f1;">
```

Comment:
For this file, could we somehow achieve the existing frontend with markdown? Reason being that this is a `.md` file and it could perhaps be easier to edit if it is in markdown if possible, what do you think @EltonGohJH?

Also, a minor nit for the current HTML: the `<body>` tag used here may not be appropriate. See screenshot below (the page ends up having two body tags, which might not be very typical:
![image](https://user-images.githubusercontent.com/41845017/188170463-c05b01c7-ed66-4edc-8507-c0ab90f725d5.png)

Reply:
@tlylt 
I updated the page to make it responsive, you can check Netlify.  I tried to use more of markdown/markbind syntax. Do tell me if that is good enough.
I already have the a simple guide on the UG, is that sufficient.

Reply:
> I already have the a simple guide on the UG, is that sufficient.

Are you referring to the following point I made?
> could we implement this in our UG/DG as well to eat our own dog food?

Sorry if it was unclear, I meant to say --> Can we implement the 404 page logic in our user guide and developer guide documentation, such that if there's a 404, it should show the custom 404 page? On top of trying this out ourselves, it also helps testing since we should be able to directly check the outcome of your changes on our PR preview.

Reply:
@tlylt 
I have updated. You should be able to see the 404 in the preview now.

Reply:
@EltonGohJH https://deploy-preview-1955--markbind-master.netlify.app/devguide/333
Seems to have some style issue when I try to visit the 404 page in DG preview

Reply:
@tlylt 
This is because the html generated uses relative pathing to the css file.
Eg.
`    <link rel="stylesheet" href="markbind/css/bootstrap.min.css">`

Is there a reason why we dun use non relative pathing.

So generate this instead
`    <link rel="stylesheet" href="/markbind/css/bootstrap.min.css">`

Reply:
I don't think there's a specific reason, it seems to have always been the case since we assumed pages are located on the filesystem where they are served too.

Reply:
Made a quick change [here](https://github.com/MarkBind/markbind/pull/2032/files).

See if that works and we can get this part unblocked 🙂

Reply:
@EltonGohJH  In case you missed this, this shouldn't pose any issues now.